hal/ia32/init.c

```diff
@@ -0,0 +1,260 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * Configuration detection and other initialisation routines
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author; Andrzej Stalke
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "../string.h"
+#include "../cpu.h"
```

Comment:
Do not reintroduce relative paths
```suggestion
#include "hal/string.h"
#include "hal/cpu.h"
```